mne_bids/mne_bids.py

```diff
@@ -655,10 +656,24 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
                          % (''.join(ALLOWED_EXTENSIONS), ext))
 
     # Copy the imaging data files
-    # Re-save FIF files to fix the file pointer for files with multiple parts
-    # This is WIP, see: https://github.com/mne-tools/mne-python/pull/5470
     if ext in ['.fif']:
-        raw.save(raw_file_bids, overwrite=overwrite)
+        n_rawfiles = len(read_raw_fif(raw_fname).filenames)
```

Comment:
So I didn't notice this when the PR was going through, but why do you re-load the raw object? Surely this wastes a bit of time, especially since we already have `raw`.
This line should be
`n_rawfiles = len(raw.filenames)`
and we can drop the additional `read_raw_fif` on line 20...
Or am I missing something???

Reply:
good point ...